File: strings/testCmp/strcmp.java
import java.util.Scanner;

public class strcmp {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str1;
        String str2;
        str1 = sc.nextLine();
        str2 = sc.nextLine();
        long t1 = System.currentTimeMillis();
            str1.equals(str2);
        long t2 = System.currentTimeMillis();
        long t3 = t2 - t1;
        System.out.println(t3);
        sc.close();
    }} /*
    Scanner sc = new Scanner(System.in);
    String str1;
    String str2; 
    {
    str1 = sc.nextLine();
    str2 = sc.nextLine();
    for (int i = 0; i < str1.length(); i++) {
        if (str1.charAt(i) == str2.charAt(i)) {
            System.out.println("As strinsgs são iguais");
        } else {
            System.out.println("As strings são diferentes");
            break;
        }
    }
    sc.close();
}
} */

File: strings/testCmp/strcmp.py
import time

def strcmp():
    str1 = input("Insira o arquivo da primeira string: ")
    str2 = input("Insira o arquivo da segunda string: ")
    
    t1 = time.time()
    for i in range(0, len(str1)):
        if str1[i] == str2[i]:
            print("As strings são iguais")
        else:
            print("As strings são diferentes")
            break
    t2 = time.time()
    print(t2 - t1)